pyanaconda/ui/cockpit/anaconda-webui/test/run-test

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+set -ex
+
+clean_up () {
+    ARG=$?
+    pkill -9 -f  'python -m http.server'
+    virsh -c qemu:///session destroy test-updates
+    virsh -c qemu:///session undefine test-updates
+    exit $ARG
+}
+
+trap clean_up SIGINT SIGTERM EXIT
+pushd ../../../../
+wget -nc https://kojipkgs.fedoraproject.org//packages/cockpit/259/1.fc35/x86_64/cockpit-ws-259-1.fc35.x86_64.rpm
+wget -nc https://kojipkgs.fedoraproject.org//packages/cockpit/259/1.fc35/x86_64/cockpit-bridge-259-1.fc35.x86_64.rpm
```

Comment:
These hardcoded paths are rather ugly -- they will become invalid in a few weeks. But I suppose thats' what the commit message refers to, and it'll go away. Please add a comment here.